clients/rest-client-core/src/main/java/uk/gov/justice/services/clients/core/RestClientProcessor.java

```diff
@@ -23,6 +28,7 @@
 public class RestClientProcessor {
 
     private static final String MEDIA_TYPE_PATTERN = "application/vnd.%s+json";
+    private static final String CPPID = "CPPID";
```

Comment:
Yes, I wanted to do that but it will require dependency on another module.  We should probably move that HeaderConstants class to rest-client-core or some kind of common module.  Requires some refactoring of that area.